src/site/content/en/fast/signed-exchanges/index.md

```diff
@@ -222,6 +248,90 @@ The **Preview** tab provides more information about the contents of a SXG.
 To see a SXG firsthand, visit this [demo](https://signed-exchange-testing.dev/)
 in [one of the browsers that supports SXG](#browser-compatibility)
 
+## Use Cases
```

Comment:
```suggestion
## Use cases
```
https://developers.google.com/style/capitalization#capitalization-in-titles-and-headings